src/main/java/com/laserfiche/repository/api/clients/impl/model/Entry.java

```diff
@@ -10,6 +12,8 @@
 import java.util.Objects;
 
 @javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaClientCodegen", date = "2022-08-19T17:07:43.799-04:00[America/New_York]")
+@JsonTypeInfo(use = JsonTypeInfo.Id.DEDUCTION)
+@JsonSubTypes({@JsonSubTypes.Type(Folder.class), @JsonSubTypes.Type(Shortcut.class)})
```

Comment:
Should `Document.class` also be added? Reminder that in the future `RecordSeries.class` too

Reply:
Oh, OK. I didn't know about it. Let me add both.

Reply:
I will also upodate the code generator PR.

Reply:
I just checked the code, yes, we will add `RecordSeries.class` when we have it. For now, we will just add `Document.class`.